test/fakeintake/aggregator/connectionsAggregator.go

```diff
@@ -0,0 +1,70 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+package aggregator
+
+import (
+	"fmt"
+
+	agentmodel "github.com/DataDog/agent-payload/v5/process"
+
+	"github.com/DataDog/datadog-agent/test/fakeintake/api"
+)
+
+// Connections type contain all payload from /api/v1/connections
+type Connections struct {
+	agentmodel.CollectorConnections
+}
+
+// name return connection payload name based on hostname and network ID
+func (c *Connections) name() string {
+	return c.HostName + c.NetworkId
+}
+
+// GetTags return tags connection
+func (c *Connections) GetTags() []string {
+	dns, err := c.GetDNSNames()
+	if err != nil {
+		return []string{}
+	}
+	return dns
+}
+
+// DecodeCollectorConnection return a CollectorConnections protobuf object from raw bytes
+func DecodeCollectorConnection(b []byte) (cnx *agentmodel.CollectorConnections, err error) {
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution. -->
**❓ question**
Do you need this to be public ? Could this be private ?

Reply:
Thanks, the classic was private, move to public during the dev but doesn't need to be public :)